### PR TITLE
add banner image to site repo

### LINKED ISSUE #79: Add the banner image to the example repository
Instead of loading the image from this repository, we should include the image in the example repository the learners are constructing as they work through the lesson. We can ask them to download it from this repo and upload it to theirs. Just need to add in some instructions for this at the start of the Includes episode...

### PR DESCRIPTION
**Pull request by tobyhodges**
Fixes #79 

Unfortunately the process of uploading a file to a new folder is not straightforward in GitHub, so the instructions are a bit harder to follow than I would have liked. Nevertheless, I think it much improves the readability of the lesson as a whole, and it's definitely better practice to load images from you own repository and not someone else's!

### CODE REVIEW

File: _episodes/06-includes.md [hunk 1]
```diff
@@ -74,14 +74,39 @@ the sites linked in the previous exercise aren't built with Jekyll.
...
+We're going to add a pre-made banner image to our repository,
+and it is good practice to store all image files in a common folder.
+1. click "Create new file" under the "Add file"
+dropdown on your repository homepage,
+2. in the "Name your file..." box, type `images/`.
+   The folder name should be automatically inserted in the path displayed
+   next to this box for naming the file.
+3. You can then leave the file blank and name it `.gitkeep`.
+   When you commit the changes, the `images` folder will have been
+   added to your repository.
```
Comment (sstevens2) [inline _episodes/06-includes.md]: This seems like a good way to do this but we could also have them drag-drop the image and then rename it into the images folder.  I'm not sure which way would be easier.  Thoughts?

Comment (sstevens2) [reply _episodes/06-includes.md]: This does teach `.gitkeep` which could be good.

File: _episodes/06-includes.md [hunk 2]
```diff
@@ -74,14 +74,39 @@ the sites linked in the previous exercise aren't built with Jekyll.
...
+We will be uploading our banner image to this folder in a moment
+and unfortunately GitHub does not provide a way to create a new folder
+while uploading existing files, only while creating new ones.
+When making these blank files,
+which exist only to allow the existence of their parent repository,
+it is traditional to call them `.gitkeep`.
+Now [download this banner image that we will add to our pages][banner-image] and
+upload the file to your newly-created `images` folder on GitHub:
+youc an do this by navigating into the folder and choosing
```
Comment (sstevens2) [inline _episodes/06-includes.md]:
```suggestion
you can do this by navigating into the folder and choosing
```

File: _episodes/06-includes.md [hunk 3]
```diff
@@ -100,7 +125,7 @@ at the top of your page.
 > > ## Solution
 > >
 > > ```
-> > <a href="https://YOUR_USERNAME.github.io/YOUR_REPO_NAME/"><img src="carpentries-incubator.github.io/building-websites-with-jekyll-and-github-or-gitlab/files/site_banner.png"></a>
+> > <a href="https://YOUR_USERNAME.github.io/YOUR_REPO_NAME/"><img src="/images/site_banner.png" alt="Group Website with Jekyll"></a>
```
Comment (sstevens2) [inline _episodes/06-includes.md]: 👍 for alt text

Comment (sstevens2) [reply _episodes/06-includes.md]: Maybe we should write why alt text is important?

**Review by sstevens2 — APPROVED** [review]
This looks good to me.  Some small suggestions in my review but I think it is fine without those changes too.

File: _episodes/06-includes.md (hunk 1 above)
Comment (tobyhodges) [reply _episodes/06-includes.md]: Let's leave it for now and change if people find this too confusing in the first trial run next week

File: _episodes/06-includes.md (hunk 2 above)
Comment (tobyhodges) [reply _episodes/06-includes.md]: addressed in 673ef5f

File: _episodes/06-includes.md (hunk 3 above)
Comment (tobyhodges) [reply _episodes/06-includes.md]: tried to address this in 4a2a87a

**Comment by sstevens2** [comment]
This looks great!  Love the call out on alt text

**Pull request merged**